target/linux/mpc85xx/image/p1010.mk

```diff
@@ -21,8 +21,8 @@ define Device/aerohive_br200-wp
   KERNEL_NAME := simpleImage.br200-wp
   KERNEL := kernel-bin | uImage none
   KERNEL_INITRAMFS := kernel-bin | uImage none
-  KERNEL_ENTRY := 0x1000000
-  KERNEL_LOADADDR := 0x1000000
+  KERNEL_ENTRY := 0x1500000
+  KERNEL_LOADADDR := 0x1500000
```

Comment:
Is it possible to change the  addresses only here? On many other targets you have to change the load address in uboot too and they have to match. I am not familiar with the target,

Reply:
In mxp85xx it works quite differ.

Please take a look TP-Link TL-WDR4900 v1 bootcmd:
`sf probe 0;sf read 2000000 60200 69f000;sf read 3000000 50000 10000;bootm 2000000 - 3000000`
BR200-WP loads kernel directly from NOR. 

Mpc85xx u-boot unpack (or just relocate uncompressed, like in this case) image into `KERNEL_LOADADDR` and boot from `KERNEL_ENTRY`. It loads dedicated mpc85xx loader, which unpack lzma compressed kernel into address 0x0. At the end of process, kernel is starting from address 0x0 without any arguments from u-boot (mpc85xx SimpleImage method).

I run tested it on WDR4900 and BR200-WP:
```
[    0.000000] Linux version 6.1.62 (a@b) (powerpc-openwrt-linux-musl-gcc (OpenWrt GCC 13.2.0 r24439-7e50a9f565dd) 13.2.0, GNU ld (GNU Binutils) 2.41) #0 Tue Nov 21 09:50:53 2023
[    0.000000] Hardware name: TP-Link TL-WDR4900 v1 e500v2 0x80212151 Freescale P1014
```